visualizations/frontend/other/TTable.vue

```diff
@@ -757,7 +757,6 @@ export default {
       if (!validProps) return;
       if (!this.rows) return;
 
-      this.setInternalColumns();
```

Comment:
What's the reason we are moving this call `this.setInternalColumns()` inside the `setupInternalRows` method? Is there something that we are doing on the `setupInternalRows` that affects the execution of `setInternalColumns`?
If that the case, we should try to avoid it, if we can't, can we move swap the line orders instead of moving the call to inside the other method (so that we avoid even more coupling).

```javascript
this.setupInternalRows();
this.setInternalColumns();
```

Reply:
More context, this is 100% a hack to unblock Kennan. The setInternalColumns call belongs exactly where it _was_ in init, hence why I didn't do a bigger refactor- this will either be reverted or, more likely, completely refactored when table is split up into a boring non-topcoat base table + layer handling component. Unfortunately onVisualizationInit is now being called before the row data is available (should probably be a bug ticket now that I think about it). Probably caused by the addition of a debounce in the created function in visualization_mixin in topcoat core. Additionally adding a watcher on rows is not working either- hence the hack here. It's all bad, this is just the least bad approach I can think of without having to dump a bunch more time into a solution. 🤷‍♀️